### PR TITLE
Changing repo URL has been ignored

### PR DESCRIPTION
**Pull request by schubi2**

### CODE REVIEW

File: src/modules/SourceDialogs.rb [hunk 1]
```diff
@@ -2540,7 +2540,11 @@ def EditDisplayInt(repository)
       Builtins.y2milestone("Ret: %1", ret)
       UI.CloseDialog
 
-      ret == :ok ? GetURL() : ""
+      if ret == :ok || ret == :next
```
Comment (dgdavid) [inline src/modules/SourceDialogs.rb]: Maybe you can change this by

```ruby
[:ok, :next].include?(ret) ? GetURL() : ""
```
to make Rubocop happy

Comment (dgdavid) [reply src/modules/SourceDialogs.rb]: I mean, to use less lines and keep the class below the 2000 lines.

Comment (dgdavid) [reply src/modules/SourceDialogs.rb]: or even

```ruby
return GetURL() if ret == :ok || ret == :next

""
```

:see_no_evil:

Comment (schubi2) [reply src/modules/SourceDialogs.rb]: I like the first one :-)

**Pull request merged**